plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -235,7 +249,11 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer, L
         activateLocationComponent()
       }
     }
-    locationPuckManager?.updateSettings(internalSettings)
+    if (!internalSettings.enabled) {
```

Comment:
nit: can we invert logic here? 
`if (internalSettings.enabled) { ... } else { ... } `
just looks better imho